src/templates/project.js

```diff
@@ -89,6 +95,11 @@ const Project = props => {
     const childNodes = getContent(dlv(content, 'children', []));
     const { siteUrl } = useSiteMetadata();
     const projectUrl = `${siteUrl}${props.pageContext.slug}`;
+    const breadcrumb = [
```

Comment:
Similar thought, should we move the STUDENT_SPOTLIGHT_BREADCRUMBS into the component to avoid the redeclaration?